### PR TITLE
chore: Update pre-commit instructions

### PR DESCRIPTION
**Pull request by raffaellasuardini**
Inside documentation now there are instructions to install pre-commit

## Issue

* #310

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,7 +128,15 @@ This is an example of how to list things you need to use the software and how to
    ```sh
    venv\Scripts\activate
    ```
-3. Create a posgresql database
+3. Install pre-commit
+   ```sh
+   pip install pre-commit
```
Comment (tim-schilling) [inline README.md:133]: The step to install requirements involves installing pre-commit. So we could put `pre-commit install` after the `pip install -r` step and maintain the number of steps.

Comment (tim-schilling) [reply README.md:133]: To clarify, `pre-commit` is in our dev requirements file.

File: README.md [hunk 2]
```diff
@@ -148,12 +140,15 @@ This is an example of how to list things you need to use the software and how to
...
    ```
-5. install requirements:
+4. install requirements:
    ```sh
    pip install -r requirements/requirements-dev.txt
    ```
-
-6. Copy `.env.template.local` file, rename to `.env` and use variables for your local postgres database.
+   set up the git hook scripts
+   ```sh
```
Comment (raffaellasuardini) [inline README.md:148]: I left a description of the instructions for clarity, is it better to remove them?

Comment (tim-schilling) [reply README.md:148]: Nope that works. Thank you

**Review by tim-schilling — APPROVED** [review]

**Pull request merged**